packages/nlopt-ocaml/nlopt-ocaml.0.5.2/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+maintainer: "michal.kurcewicz@gmail.com"
+authors: ["Michał Kurcewicz"]
+homepage: "https://bitbucket.org/mkur/nlopt-ocaml"
+bug-reports: "https://github.com/mkur/nlopt-ocaml/issues"
+dev-repo: "git+https://github.com/mkur/nlopt-ocaml.git"
+license: "LGPL-2.1+ with OCaml linking exception"
+build: [
+  ["oasis" "setup"]
+  ["ocaml" "setup.ml" "-configure" "--prefix" prefix]
+  ["ocaml" "setup.ml" "-build"]
+]
+remove: [["ocamlfind" "remove" "nlopt"]]
+depends: [
+  "ocaml" {>= "3.12"}
+  "ocamlfind"
```

Comment:
```suggestion
  "ocamlfind" {build}
```
also the source code doesn't seem to make any use of `ocamlfind` outside the build phase. Could you add the the `{build}` attribute if appropriate?